src/pages/transaction-history/transaction-history.page.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from "react";
+import {
+  Divider,
+  Grid,
+  CircularProgress,
+  Paper,
+  Box,
+  TableContainer,
+  Table,
+  TableBody,
+  TableHead,
+  TableRow,
+  TableFooter,
+  TablePagination,
+} from "@mui/material";
+import { users } from "data/user";
+import { StyledTableCell, StyledTableRow } from "../../components";
+
+export const TransactionHistory = () => {
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(5);
+
+  const handleChangePage = (
+    event: React.MouseEvent<HTMLButtonElement> | null,
+    newPage: number
+  ) => {
+    setPage(newPage);
+  };
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  const [transactions, setTransactions] = useState([
+    {} as {
+      transactionId: string;
+      transactionAmount: number;
+      transactionAt: Date;
+      transactionType: string;
+      transactionAccountNumber: string;
+      transactionAccountTitle: string;
+      balanceAfterTransaction: number;
+    },
+  ]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      setLoading(false);
+      setTransactions(users[0].transactions.reverse());
```

Comment:
there is library that Abdul Basit is using to fetch data from constants file, it mocks api logic and flow. I think if you can't implement the backend, maybe we can use that instead?

Reply:
I have replaced all this with context and useReducer in the next PR and that acts like a global state. Won't need to use a library.

Reply:
That's for backend logic